### ISSUE TITLE
WCAG Techniques "Official version" link appears to be incorrect

### ISSUE THREAD

**Issue by jribbens** [w3c/wcag#2571]
I mentioned several problems with the "Official version" link for WCAG 2.1 in #1740, which was unfortunately closed with only some of them being fixed. The remaining issues are:

* The text of the 'Official link' in the header of the page says `https://www.w3.org/WAI/WCAG21/techniques/`, i.e. with a lower-case `t`, whereas the actual official link appears to have a capital `t`.
* The address _linked to_ by that same text is not the same as the URL indicated by the text: it links to `https://www.w3.org/WAI/WCAG21/Understanding/`, which is a completely different document.

And come to think of it:

* The 'Editor's Draft' link has the same link/text mismatch problem.
* The WCAG 2.2 Techniques 'Official link' has the same capital-t/lowercase-t problem.

**Comment by mbgower** [comment]
> The text of the 'Official link' in the header of the page says https://www.w3.org/WAI/WCAG21/techniques/, i.e. with a lower-case t, whereas the actual official link appears to have a capital t.

I'm not sure if I'm following what you mean by "the 'Official link' in the header of the page"
I'm showing a screen snippet which shows the top of the page. The URL reads "Techniques"; the banner reads "Techniques"; the heading reads "Techniques"
<img width="1178" alt="image" src="https://github.com/w3c/wcag/assets/14298245/af0cfe7c-6114-4f01-81f6-d2d52fd88669">

My inclination is to conclude that the previous problem must be resolved. Correct me if I'm wrong, and please provide a more exact description of what you are flagging.

Similarly, I cannot identify which url points to the Understanding home page. I did a page source search and could not locate. I am assuming it has also been resolved.

Finally, I also cannot find these issues on the 2.2 site.

I am concluding these have been resolved, and so am closing the issue. If you do not believe the issue has been resolved, please reopen it and add clarifying comments.
Thanks!

**Comment by jribbens** [comment]
@mbgower Yes, it looks like the page was significantly redesigned a few months after I opened this issue, in commit 6aa6742a2cf9f05fcc16ccc219510df508a092e9, and the content which I was referring to has been completely removed. So you are right to close the issue.
